src/test/unit_tests/jest_tests/test_nc_nsfs_account_cli.test.js

```diff
@@ -1100,6 +1100,23 @@ describe('manage nsfs cli account flow', () => {
                 const account = await config_fs.get_account_by_name(name, config_fs_account_options);
                 expect(account.nsfs_account_config.supplemental_groups).toStrictEqual(expected_groups);
             });
+
+            it('cli account update - cli update account with empty supplemental groups (unset)', async function() {
```

Comment:
what will happen if there is one supplemental groups  added by admin and we send account update without any supplemental groups. could you please add that test

Reply:
if you don't add this option at all, it should remain the same. I'll add a test for it